monai/deploy/utils/deviceutil.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+
+
+def has_rocm():
+    """Return True if ROCm is installed and GPU device is detected.
+
+    Args:
+
+    Returns:
+        True if ROCm is installed and GPU device is detected, otherwise False.
+    """
+    cmd = "rocminfo"
```

Comment:
The command errors out when no ROCm installed. So the method should have a try/except block on calling the rocminfo command, and return false on except.

Reply:
Do the new commit address these issues?